samples/system-test/inspect.test.js

```diff
@@ -120,11 +150,43 @@ it.skip('should inspect multiple GCS text files', async () => {
   );
   assert.strictEqual(
     new RegExp(/Found \d instance\(s\) of infoType PHONE_NUMBER/).test(output),
-    true
+    true,
+    `Actual output was:\n ${output}`
   );
   assert.strictEqual(
     new RegExp(/Found \d instance\(s\) of infoType EMAIL_ADDRESS/).test(output),
-    true
+    true,
+    `Actual output was:\n ${output}`
+  );
+});
+
+it.skip('should inspect GCS text files with sampling', async () => {
```

Comment:
Guessing this will get cleaned up, but making sure to point out the new skipped tests

Reply:
I don't have access to the nodejs-docs-samples-dlp bucket. I tried them out on one of my own buckets and they worked, but I couldn't run them as-is without bucket access, so I kept them as skipped.